backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/GraphQLContext.kt

```diff
@@ -2,9 +2,11 @@ package app.ehrenamtskarte.backend.common.webservice
 
 import app.ehrenamtskarte.backend.auth.webservice.JwtPayload
 import com.expediagroup.graphql.execution.GraphQLContext
+import javax.servlet.http.Part
 
 data class GraphQLContext(
-    val jwtPayload: JwtPayload?
+    val jwtPayload: JwtPayload?,
+    val files: List<Part>
```

Comment:
Are we sure that the InputStream within Part is closed eventually if a part is not retrieved?

Reply:
I'm relatively sure, that javalin takes care of that.